src/components/molecules/Visualizer/hooks.ts

```diff
@@ -161,6 +161,10 @@ export default ({
     engineRef.current?.requestRender();
   });
 
+  const onInfoboxMaskClick = useCallback(() => {
+    selectLayer(undefined);
+  }, [selectLayer]);
+
```

Comment:
Since other functions' names start `handle`, it's better to use the same convention. 

```suggestion
  const handleInfoboxMaskClick = useCallback(() => {
    selectLayer(undefined);
  }, [selectLayer]);
  
```